__tests__/server/utils/onModuleLoad.spec.jsx

```diff
@@ -326,6 +330,21 @@ describe('onModuleLoad', () => {
     expect(configurePWA).toHaveBeenCalledWith(pwa);
   });
 
+  it('calls setErrorPage with error page URL', () => {
+    const errorPageUrl = 'https://example.com';
+    onModuleLoad({
+      module: {
+        [CONFIGURATION_KEY]: {
+          csp,
+          errorPageUrl,
+        },
+        [META_DATA_KEY]: { version: '1.0.14' },
+      },
+      moduleName: 'some-root',
+    });
+    expect(setErrorPage).toHaveBeenCalledWith(errorPageUrl);
```

Comment:
We probably need to look into the negative scenario where the `errorPage` doesn't load / the url is invalid or the content is too large. We don't want the app to crash if something like that happens when the rootModule is loaded, we should just add a log to warn that the error page wasn't loaded and carry on

Reply:
@infoxicator this follows everything else that fails when the root module loads. If your `csp` isn't provided we cause everything to _crash_/kill the server. @mtomcal said the same idea should be applied here that if the URL that is fetched isn't correct to kill the server. If you have a broken error page or if its too large we don't want the app starting up.

Reply:
To add that @infoxicator, consider a new Root Module version (e.g. v2.0.0) is loaded to a running One App with a stable Root Module (e.g. v1.0.0). A Root Module v2.0.0 with an invalid error page url, will cause the Root Module v2.0.0 to not get loaded and fallback to original v1.0.0 until a fix v2.1.0 module is corrected. We do no crash the server if it exists in a previously stable state. As @JamesSingleton mentions, this follows the same pattern as how CSP behaves on loading of a new Root Module.

Reply:
@mtomcal thanks for confirming, so if there is a previous working version that should be fine 👍